MM2RandoLib/Resources/Asm/config.asm

```diff
@@ -0,0 +1,77 @@
+.macpack common
+
+.segment "HEADER" :bank $00 :size $0010 :mem $0000 :off $00000
```

Comment:
Could you add comments to this. I can't really tell what it does by looking at it.

Reply:
Yeah.